Firmware/ultralcd.cpp

```diff
@@ -4726,15 +4728,15 @@ static void lcd_settings_menu()
     MENU_ITEM_SUBMENU_P(_T(MSG_FSENSOR), lcd_fsensor_settings_menu);
 #endif //FILAMENT_SENSOR
 
-	SETTINGS_SPOOLJOIN;
-
-	SETTINGS_CUTTER;
-
     MENU_ITEM_TOGGLE_P(PSTR("MMU"), eeprom_read_byte((uint8_t *)EEPROM_MMU_ENABLED) ? _T(MSG_ON) : _T(MSG_OFF), mmu_enable_switch);
 
-    if (eeprom_read_byte((uint8_t *)EEPROM_MMU_ENABLED))
+    if (MMU2::mmu2.Enabled())
```

Comment:
:+1: 

but the Reset MMU option should be probably available even when the MMU is not communicating